include/cufinufft/impl.h

```diff
@@ -311,13 +316,16 @@ int cufinufft_makeplan_impl(int type, int dim, int *nmodes, int iflag, int ntran
                  d_fseries_precomp_phase.begin());
     thrust::copy(fseries_precomp_f, fseries_precomp_f + 3 * MAX_NQUAD,
                  d_fseries_precomp_f.begin());
+    if(!d_plan->opts.gpu_spreadinterponly) {
```

Comment:
This is a good idea to skip this if you don't need it. But, fwkerhalf1 etc are still allocated, right?

Reply:
I am not sure they are. Lemme point and ask as a question in this PR.